plugins/adminhelp.sma

```diff
@@ -13,133 +13,181 @@
 
 #include <amxmodx>
 
-new g_cvarDisplayClientMessage;
-new g_cvarHelpAmount;
+const MaxMapLength         = 32;
```

Comment:
It's odd indeed, and oddly works well as far as I can tell, it downloaded the map correctly and I can play correctly. If I remember from memory, the important parts are [40] or [64] since they include ```maps/``` and ```.bsp```